confetti/src/main/java/com/github/jinatonic/confetti/confetto/Confetto.java

```diff
@@ -70,6 +65,77 @@
     private float overrideX, overrideY;
     private float overrideDeltaX, overrideDeltaY;
 
+    // Visible for testing
+    protected static Long computeMillisToReachTarget(Float targetVelocity, float initialVelocity,
```

Comment:
this is inconsistent with the style of rest of the repo, use double tabs for overflow lines.

Reply:
this method is also not changed, please revert the change and move it back to its original location. don't want to unnecessarily modify git blame.